CMSIS/Core_A/Include/core_ca.h

```diff
@@ -783,6 +783,175 @@ typedef struct
 
 #define GICDistributor      ((GICDistributor_Type      *)     GIC_DISTRIBUTOR_BASE ) /*!< \brief GIC Distributor register set access pointer */
 
```

Comment:
The names if bits in register GICDistributor.CTLR change depending of the system and the security state.
Is there a possibility to get this covered in the *_Pos/*_Mask naming?

Typo for bit 31in  GICDistributor.CTLR register. RWP not RPW).

Reply:
Correct, I was also thinking about this. My first thought was to create the corresponding definitions for all possibilities. But this was not possible, because EnableGrp1 is used for bit position 0 (single security) as well as for bit position 1 (non-secure).
I have no experience with Security states so I don’t know if there is a way to determine which one is available or active. So, I decided to use only the one which I thought is the common one. If it is okay for you, I would use only the single Security state.

Typo is fixed.